lib.php

```diff
@@ -220,7 +220,7 @@ public function get_links($linkarray) {
 
                         $helpicon = $OUTPUT->pix_icon(
                             'help',
-                            get_string('submissiondisplayerror:eulanotaccepted', 'plagiarism_turnitinsim'),
+                            get_string('submissiondisplayerror:eulanotaccepted_help', 'plagiarism_turnitinsim'),
```

Comment:
Is it possible to write unit test cases for these changes?

Reply:
I've added one and just piggybacked on to the existing test for EULA not accepted.